framework/midi/midievent.h

```diff
@@ -231,15 +231,15 @@ struct Event {
         return 0;
     }
 
-    [[deprecated]] EventType type() const
+    /*[[deprecated]]*/ EventType type() const
```

Comment:
This constructor will be deleted in MU4. That's why it's marked as deprecated and must not be used.

Reply:
dito